.github/workflows/ci.yml

```
@@ -7,11 +7,14 @@ on:
 
 jobs:
   lint_test:
-    uses: babylonlabs-io/.github/.github/workflows/reusable_go_lint_test.yml@v0.1.0
+    uses: babylonlabs-io/.github/.github/workflows/reusable_go_lint_test.yml@v0.6.0
     with:
      run-unit-tests: true
      run-integration-tests: true
      run-lint: true
+     run-build: true
+     run-gosec: true
+     gosec-args: "-exclude-generated ./..."
```

```suggestion
     gosec-args: "-no-fail -exclude-generated ./..."
```

Suggesting to add the `-no-fail` flag so that gosec errors won't make the pipeline fail.
After we decide which errors we want to resolve and which we can ignore, we should remove this flag.